# Polaris v1.5 release note (August 2018)

# Table of Contents
- [Overview](#Overview)
- [Summary statistics](#truthset)
- [Validation Scheme](#validation-scheme)
- [Merging Scheme](#merging-and-refining-scheme)

## Overview

### **Polaris truth data**
The Polaris data takes structural variants (SV) input from multiple resources, perform population and pedigree validation, and finally output a VCF labeled with the filters from validation.

Variants that pass our pedigree and population check were labeled as *PASS* in their filter fields.

Variants that fail one or more filter were labled with the specific filter name(s) in their filter fields. These variants may be:

- False positive calls
- Variants with inaccurate breakpoints.
    - In the next release, breakpoint refinement will be introduced and we believe the refinement will greatly increase the validation rate of such variants
- Variants that are difficult to be genotyped with our current methods, including:
    - STRs
    - Complex SVs

### **Source of input variants**

v1.5 truth set consists SVs from:

- Manta deletion and insertion calls from NovaSeq NA12877 & NA12878

- Refined Sniffles (v1.0.8) deletion and insertion calls from PacBio NA12878 on hg38
    - Insertion sequence was assembled and refined from PacBio + ONT reads

- Copy number variants and large deletions curated from population and Platinum Genome pedigree on *hg19*, coming from deletion calls of Manta/Canvas and Sniffles.

In the release VCF, the *SOURCE* keys in *INFO* fields indicate which dataset(s) a variant was originally called.

### **Samples genotyped in truth data**

The release vcf contains genotypes of:

- Parents and grandparents in the Platinum Genome pedigree
- Ashkenazi trio
- 50 trio families in Polaris

## Summary statistics

The input variants and the validation pipeline is based on hg38. Hg19 data was generated by liftover.

12,374 (98%) passed entries and 24,405 (88%) failed entries were successfully converted to hg19.

### Merged variants partitioned by SVLEN and type

|         SV Type       |    INS    |    INS    |    DEL    |    DEL    |    CNV    |    CNV    |
|:----------------------|:---------:|:---------:|:---------:|:---------:|:---------:|:---------:|
|          Filter       |    PASS   |    FAIL   |    PASS   |    FAIL   |    PASS   |    FAIL   |
|   _L_ \< 50           |    905    |   1,934   |   1,178   |   1,390   |     0     |     0     |
|  50 \< _L_ \< 100 bp  |   1,503   |   3,571   |   1,939   |   3,553   |     0     |     0     |
|  100 \< _L_ \< 1kb    |   2,240   |   4,785   |   2,389   |   4,417   |     0     |     0     |
|  1kb \< _L_ \< 10kb   |     18    |    155    |   1,978   |   2,930   |    105    |    139    |
|   _L_ > 10kb          |     0     |     0     |    337    |   1,065   |     89    |    219    |
| __Total__             | __4,666__ | __10,445__| __7,821__ | __13,355__|  __194__  |  __358__  |

### **Non-reference** calls in NA12878 partitioned by SVLEN and type

|         SV Type       |    INS    |    DEL    |    CNV    |
|:----------------------|:---------:|:---------:|:---------:|
|   _L_ \< 50           |    878    |   1,169   |     0     |
|  50 \< _L_ \< 100 bp  |   1,274   |   1,521   |     0     |
|  100 \< _L_ \< 1kb    |   1,958   |   1,926   |     0     |
|  1kb \< _L_ \< 10kb   |     17    |    626    |     19    |
|   _L_ > 10kb          |     0     |    140    |     11    |
| __Total__             | __4,127__ | __5,382__ |   __30__  |


## Validation Scheme

Our validation pipeline consists of three major steps:
- Consistency check in the Platinum Genome pedigree
- Trio consistency check in Polaris families
- Population HWE check in our internal samples

Hard filtering is also applied after these steps.

In this validation pipeline, samples were independently genotyped using graph-genotyper *Paragraph* v2.0:

- https://github.com/Illumina/paragraph

Additionally, for large deletions and CNVs, samples were additionally genotyped with our internal depth-based caller.

### Detailed validation criteria

A PASS variant needs to pass **all** below filters.

|                  Category                   |          Pass criterion        |
|:--------------------------------------------|:------------------------------:|
|            Pedigree consistency             |     <sup>1</sup>Hamming distance <= 1      |
|           Call rate in population           |             >= 0.8             |
|     Paragraph internal filter pass rate     |             >= 0.5             |
|            trio inconsistency rate<sup>2</sup>          | <=0.1 x #kid-non-reference GTs |
|                 <sup>3</sup>HWE p value                 |            >= 0.001            |


<sup>1</sup> Hamming distance represents the number of wrong genotypes in the pedigree.

<sup>2</sup> If a variant is all alternative homozygous in the pedigree, we require it's either not all homozygous reference in the population or in the trio.

<sup>3</sup> This filter does not apply if a variant meets one of the following criteria:
    - pass pedigree consistency check (hamming distance = 0 and not all reference homozygous)
    - pass the filter of trio inconsistency rate

## Merging Scheme

Merging was only performed between PASS variants. Failed variants were not merged and were kept as what they are.

CNVs are proritized over other types of variants.

For other overlapping PASS variants:
- If the reciprocal overlap is less than 80%, do not merge.
- If the reciprocal overlap is higher than 80%, then examine the validation statistics:
    - Keep the best variant as it is
    - Label the suboptimal variants as "MERGED" at filter field.
