# Polaris

<p align="center">
<a href="../../wiki/Sample-Information">
<img src="https://img.shields.io/badge/Total%20samples%20sequenced-220-6d73f3.svg" height="30">
</a>
</p>

<p align="center">
<a href="../../wiki/Sample-Information#hiseqx-data-polaris">
<img src="https://img.shields.io/badge/HiSeqX%20data-Polaris%201-ed9d2d.svg" alt="HiSeqX data">
</a>
</p>

<p align="center">
<a href="release-notes/vc1_0.md#get-the-data">
<img src="https://img.shields.io/badge/Latest%20variant%20calls-v1.5-8a6183.svg" alt="HiSeqX data">
</a>
</p>

<!-- [More information!][1.1] -->

[TOC]: #

# Table of Contents
- [Summary](#summary)
- [Variant calls](#variant-calls)
    - [Download the VCF](#download-the-vcf)
- [Sequencing resources](#sequencing-resources)
    - [HiSeqX PCR-Free Data (Polaris 1)](#hiseqx-pcr-free-data-polaris-1)
    - [Associated resources](#associated-resources)
        - [Platinum Genomes](#platinum-genomes)
        - [HiSeqX PCR-Free](#hiseqx-pcr-free)
    - [Coming soon](#coming-soon)
        - [HiSeqX PCR-Free](#hiseqx-pcr-free)
        - [10X](#10x)
        - [NovaSeq S4 PCR-Free](#novaseq-s4-pcr-free)
- [Issues](#issues)
- [References](#references)


## Summary

The Polaris project provides
* Population sequencing resources on high throughput Illumina sequencing platforms
* Variant calls from multiple technologies, validated by population genetics and Mendelian methods

Further details of the sequencing resources, input data sources, genotyping
methods and validation methods can be found in the [project wiki][1.1].

## Variant calls

Our latest variant call release set is v1.5, containing breakpoint-resolved structural variants (SVs) from:

- Manta deletion and insertion calls from NovaSeq S1 NA12877 and NA12878

- Refined Sniffles simple SV calls from PacBio NA12878 on hg38
    - Based on the calls generated at [Sedlazeck et al. 2018](https://www.nature.com/articles/s41592-018-0001-7), insertion sequence was re-assembled and refined from PacBio and Oxford Nanopore reads.

- Copy number variants and large deletions called from population and Platinum Genome pedigree using our [depth-based CNV caller](wiki/Joint-Genotyping-Methods#depth-based-cnv-caller)

Please check our [release-notes/v1.5](release_notes/v1.5) for details about validation.

In total, there are 13,120 PASS and 27,093 FAIL entries on **hg38**. Hg19 calls were generated by liftover.

### Download the VCF

VCFs for hg38 and hg19 are available at [release-data/v1.5](release-data/v1.5)

## Sequencing resources

Population panels with unrestricted access sequenced as part of Polaris are
available through [BaseSpace][3.1] or the
[European Nucleotide Archive (ENA)][3.2].

Additional panels are available through the [EGA][3.3] or [dbGaP][3.4] with
restricted access subject to approval through a Data Access Committee. **No
variant calls are ever reported in Polaris for restricted access panels.**

Further information the sequencing resources described below can be found in the
[project wiki][0.3].

### HiSeqX PCR-Free Data (Polaris 1)

All HiSeqX PCR-Free data was generated by Illumina Laboratory Services (ILS)
with a target whole genome coverage of 30X.

Here are four unrestricted access panels available in Polaris:

1. **[Diversity panel][3.1.1.1]** ([BaseSpace][3.1.1.2], [ENA][3.1.1.3]) &mdash;
   150 samples selected to represent a diversity of populations
2. **[PGx panel][3.1.2.1]** (BaseSpace (pending), [ENA][3.1.2.3]) &mdash; 70
   samples with orthogonally validated genotypes for 28 genes relevant for
   PGx<sup>[4](#Pratt2016)</sup>
3. **[Kids panel][3.1.3.1]** (previously named as "trio panel"), [ENA][3.1.3.2] &mdash; 51 children whose parents were sequenced as part of the diversity panel
4. **PGx 10X® panel**, [ENA][3.1.3.4] &mdash; the same 70 samples from the PGx panel, prepared with the 10X Genomics® Chromium Controller™ and sequenced on the HiSeq® 4000

There is also a restricted access [repeat expansion panel][3.1.4.1] available
through [EGA][3.1.4.2].

### Associated resources

#### [Platinum Genomes][3.5]

##### HiSeq2000 PCR-free

* **Parents & grandparents**
  * [ENA][3.2.1]
  * [BaseSpace][3.2.2]
* **Children**
  * [dbGaP][3.2.3]

#### HiSeqX PCR-Free

* **Parents & grandparents**
  * *ENA* &mdash; pending
  * *BaseSpace* &mdash; pending
* **Children**
  * *dbGaP* &mdash; pending

#### 10X

* Platinum Genomes Pedigree

### Coming soon

#### HiSeqX PCR-Free

* Platinum Genomes pedigree
* NIST Ashkenazi Jewish trio

#### NovaSeq S4 PCR-Free

* Platinum Genomes pedigree
* NIST Ashkenazi Jewish trio

## Issues

Please [open an issue][4.1] to provide feedback or ask questions.

## References

1. <a name="Eberle2017"></a>Eberle, et al (2017) A reference data set of 5.4
   million phased human variants validated by genetic inheritance from
   sequencing a three-generation 17-member pedigree. *Genome Res.* 27:157-164.
   [doi:10.1101/gr.210500.116][5.2]
2. <a name="English2015"></a>English, et al (2015) Assessing structural
   variation in a personal genome-towards a human reference diploid genome. *BMC
   Genomics.* 16:286 [doi:10.1186/s12864-015-1479-3][5.4]
3. <a name="Kehr2017"></a>Kehr, et al (2017) Diversity in non-repetitive human
   sequences not found in the reference genome. *Nat Genet.* 49(4):588-593.
   [doi: 10.1038/ng.3801][5.3]
4. <a name="Pratt2016"></a> Pratt, et al (2016) Characterization of 137 Genomic
   DNA Reference Materials for 28 Pharmacogenetic Genes: A GeT-RM Collaborative
   Project. *J Mol Diagn.* 18(1):109-23. [doi:10.1016/j.jmoldx.2015.08.005][5.1]
5. <a name="Pratt2016"></a> Sedlazeck, et al (2018) Accurate detection of complex structural variations using single-molecule sequencing. *Nat Method.* 15:461-468.

[0.1]: ../../wiki/Sample-Information#hiseqx-data-polaris-1
[1.1]: ../../wiki/
[2.1]: release-notes/vc1_0.md
[2.2.1]: ../..//wiki/Input-Data-Sources#pop-manta
[2.2.2]: ../..//wiki/Input-Data-Sources#pg-pop
[2.2.3]: ../../wiki/Input-Data-Sources#parliament-insertions
[2.2.4]: ../..//wiki/Input-Data-Sources#popins-icelandic-insertions
[2.3]: ../../wiki/Joint-Genotyping-Methods#breakpoint-resolved-structural-variant-calls
[2.4.1]: https://github.com/Illumina/PlatinumGenomes
[2.5.1]: https://illumina.github.io/Polaris/
[2.5.2]: https://aws.amazon.com/cli/
[3.1]: https://basespace.illumina.com/home/index
[3.2]: https://www.ebi.ac.uk/ena
[3.3]: https://ega-archive.org/
[3.4]: https://www.ncbi.nlm.nih.gov/gap
[3.1.1.1]: ../../wiki/HiSeqX-Diversity-Panel
[3.1.1.2]: https://euc1.sh.basespace.illumina.com/projects/2265263
[3.1.1.3]: https://www.ebi.ac.uk/ena/data/view/PRJEB20654
[3.1.2.1]: ../../wiki/HiSeqX-PGx-Panel
[3.1.2.3]: https://www.ebi.ac.uk/ena/data/view/PRJEB19931
[3.1.2.4]: https://www.ebi.ac.uk/ena/data/view/PRJEB25009
[3.1.3.1]: ../../wiki/HiSeqX-Trio-Panel
[3.1.3.2]: https://www.ebi.ac.uk/ena/data/view/PRJEB25009
[3.1.3.4]: https://www.ebi.ac.uk/ena/data/view/PRJEB26950
[3.1.4.1]: ../../wiki/HiSeqX-Repeat-Expansion-Panel
[3.2.1]: https://www.ebi.ac.uk/ena/data/view/PRJEB3381
[3.2.2]: https://basespace.illumina.com/s/2K7LqNG7Mt1h
[3.2.3]: https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001224.v1.p1
[3.5]: ../../wiki/Sample-Information#platinum-genomes
[4.1]: issues
[5.1]: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4695224/
[5.2]: http://genome.cshlp.org/content/27/1/157
[5.3]: https://www.nature.com/ng/journal/v49/n4/full/ng.3801.html
[5.4]: https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-015-1479-3
