# Polaris

<p align="center">
<a href="../../wiki/Sample-Information">
<img src="https://img.shields.io/badge/Total%20samples%20sequenced-270-6d73f3.svg" height="30">
</a>
</p>

<p align="center">
<a href="../../wiki/Sample-Information#hiseqx-data-polaris-1">
<img src="https://img.shields.io/badge/HiSeq%20X%20data-Polaris%201-ed9d2d.svg" alt="HiSeq<sup>&trade;</sup> X data">
</a>
</p>

<p align="center">
<a href="release-data/v2.0/download-instructions.txt">
<img src="https://img.shields.io/badge/Latest%20variant%20calls-v2.0-8a6183.svg" alt="HiSeq<sup>&trade;</sup> X data">
</a>
</p>

<!-- [More information!][1.1] -->

[TOC]: #

# Table of Contents
- [Summary](#summary)
- [Variant calls](#variant-calls)
    - [Download the VCF](#download-the-vcf)
- [Sequencing resources](#sequencing-resources)
    - [HiSeq X PCR-Free Data (Polaris 1)](#hiseq-x-pcr-free-data-polaris-1)
    - [Associated resources](#associated-resources)
        - [Platinum Genomes](#platinum-genomes)
        - [HiSeq X PCR-Free](#hiseq-x-pcr-free)
    - [Pending cohorts](#pending-cohorts)
        - [HiSeq X PCR-Free](#hiseq-x-pcr-free)
        - [10X Chromium](#10x-chromium)
        - [NovaSeq 6000 S4 PCR-Free](#novaseq-6000-s4-pcr-free)
- [Issues](#issues)
- [References](#references)


## Summary

The Polaris project provides
* Population sequencing resources on high throughput Illumina sequencing platforms
* Variant calls from multiple technologies, validated by population genetics and Mendelian methods

Further details of the sequencing resources, input data sources, genotyping
methods and validation methods can be found in the [project wiki][1.1].

## Variant calls

Our latest variant call truth set of Structural Variants (SVs) is v2.0. Please check our [release-notes/v2.0](release-notes/v2.0/README.md) for details.

### Download the VCF

To download the data, please do:

**Genome version *hg38***
```
wget https://s3-us-west-1.amazonaws.com/illumina-polaris-v2.0-release-data/all.merge.hg38.vcf.gz
wget https://s3-us-west-1.amazonaws.com/illumina-polaris-v2.0-release-data/all.merge.hg38.vcf.gz.tbi
```
**Genome version *hg19***
```
wget https://s3-us-west-1.amazonaws.com/illumina-polaris-v2.0-release-data/all.merge.hg19.vcf.gz
wget https://s3-us-west-1.amazonaws.com/illumina-polaris-v2.0-release-data/all.merge.hg19.vcf.gz
```
## Sequencing resources

Population cohorts with unrestricted access sequenced as part of Polaris are
available through [BaseSpace][3.1], the
[European Nucleotide Archive (ENA)][3.2], and the [Sequence Read Archive][3.5].

Additional cohorts are available through the [EGA][3.3] or [dbGaP][3.4] with
restricted access subject to approval through a Data Access Committee. **No
variant calls are ever reported in Polaris for restricted access cohorts.**

Further information the sequencing resources described below can be found in the
[project wiki][0.3].

### HiSeq<sup>&trade;</sup> X PCR-Free Data (Polaris 1)

All HiSeq<sup>&trade;</sup> X PCR-Free data was generated by Illumina Laboratory
Services (ILS) with a target whole genome coverage of 30X.

There are currently four unrestricted access cohorts available in Polaris:

1. **[Diversity Cohort][3.1.1.1]** ([BaseSpace][3.1.1.2], [ENA][3.1.1.3],
   [SRA][3.1.1.4]) &mdash; 150 samples selected to represent a diversity of
   populations
2. **[Kids Cohort][3.1.3.1]** (BaseSpace, [ENA][3.1.3.3], [SRA][3.1.3.4])
   &mdash; 50 children whose parents were sequenced as part of the Diversity
   cohort
3. **[PGx Cohort][3.1.2.1]** (BaseSpace, [ENA][3.1.2.3], [SRA][3.1.2.4]) &mdash;
   70 samples with orthogonally validated genotypes for 28 genes relevant for
   PGx<sup>[4](#Pratt2016)</sup>
4. **[PGx 10X<sup>&copy;</sup> Cohort][3.1.2.1]** ([ENA][3.1.2.1.1],
   [SRA][3.1.2.1.2]) &mdash; the same 70 samples from the PGx cohort, prepared
   with the 10X Genomics Chromium Controller<sup>&trade;</sup>
   and sequenced on the HiSeq<sup>&trade;</sup> 4000

There is also a restricted access [repeat expansion cohort][3.1.4.1] available
through [EGA][3.1.4.2].

### Associated resources

#### [Platinum Genomes][3.6]

##### HiSeq<sup>&trade;</sup> 2000 PCR-free

* **Parents & grandparents**
  * [ENA][3.2.1]
  * [BaseSpace][3.2.2]
* **Children**
  * [dbGaP][3.2.3]

#### HiSeq<sup>&trade;</sup> X PCR-Free

* **Parents & grandparents**
  * *ENA* &mdash; pending
  * *BaseSpace* &mdash; pending
* **Children**
  * *dbGaP* &mdash; pending

### Pending cohorts

#### HiSeq<sup>&trade;</sup> X PCR-Free

* Platinum Genomes pedigree
* NIST Ashkenazi Jewish trio

#### 10X<sup>&copy;</sup> Chromium

* Platinum Genomes Pedigree

#### NovaSeq<sup>&trade;</sup> 6000 S4 PCR-Free

* Platinum Genomes pedigree
* NIST Ashkenazi Jewish trio

## Issues

Please [open an issue][4.1] to provide feedback or ask questions.

## References

1. <a name="Eberle2017"></a>Eberle, et al (2017) A reference data set of 5.4
   million phased human variants validated by genetic inheritance from
   sequencing a three-generation 17-member pedigree. *Genome Res.* 27:157-164.
   [doi:10.1101/gr.210500.116][5.2]
2. <a name="English2015"></a>English, et al (2015) Assessing structural
   variation in a personal genome-towards a human reference diploid genome. *BMC
   Genomics.* 16:286 [doi:10.1186/s12864-015-1479-3][5.4]
3. <a name="Kehr2017"></a>Kehr, et al (2017) Diversity in non-repetitive human
   sequences not found in the reference genome. *Nat Genet.* 49(4):588-593.
   [doi: 10.1038/ng.3801][5.3]
4. <a name="Pratt2016"></a> Pratt, et al (2016) Characterization of 137 Genomic
   DNA Reference Materials for 28 Pharmacogenetic Genes: A GeT-RM Collaborative
   Project. *J Mol Diagn.* 18(1):109-23. [doi:10.1016/j.jmoldx.2015.08.005][5.1]
5. <a name="Pratt2016"></a> Sedlazeck, et al (2018) Accurate detection of complex structural variations using single-molecule sequencing. *Nat Method.* 15:461-468.

[0.1]: ../../wiki/Sample-Information#hiseqx-data-polaris-1
[1.1]: ../../wiki/
[2.1]: release-notes/vc1_0.md
[2.2.1]: ../..//wiki/Input-Data-Sources#pop-manta
[2.2.2]: ../..//wiki/Input-Data-Sources#pg-pop
[2.2.3]: ../../wiki/Input-Data-Sources#parliament-insertions
[2.2.4]: ../..//wiki/Input-Data-Sources#popins-icelandic-insertions
[2.3]: ../../wiki/Joint-Genotyping-Methods#breakpoint-resolved-structural-variant-calls
[2.4.1]: https://github.com/Illumina/PlatinumGenomes
[2.5.1]: https://illumina.github.io/Polaris/
[2.5.2]: https://aws.amazon.com/cli/
[3.1]: https://basespace.illumina.com/home/index
[3.2]: https://www.ebi.ac.uk/ena
[3.3]: https://ega-archive.org/
[3.4]: https://www.ncbi.nlm.nih.gov/gap
[3.5]: https://www.ncbi.nlm.nih.gov/sra
[3.1.1.1]: ../../wiki/HiSeqX-Diversity-Cohort
[3.1.1.2]: https://euc1.sh.basespace.illumina.com/projects/2265263
[3.1.1.3]: https://www.ebi.ac.uk/ena/data/view/PRJEB20654
[3.1.1.4]: https://www.ncbi.nlm.nih.gov/bioproject/387148
[3.1.2.1]: ../../wiki/HiSeqX-PGx-Cohort
[3.1.2.3]: https://www.ebi.ac.uk/ena/data/view/PRJEB19931
[3.1.2.4]: https://www.ncbi.nlm.nih.gov/bioproject/385935
[3.1.2.1.1]: https://www.ebi.ac.uk/ena/data/view/PRJEB26950
[3.1.2.1.2]: https://www.ncbi.nlm.nih.gov/bioproject/474329
[3.1.3.1]: ../../wiki/HiSeqX-Kids-Cohort
[3.1.3.3]: https://www.ebi.ac.uk/ena/data/view/PRJEB25009
[3.1.3.4]: https://www.ncbi.nlm.nih.gov/bioproject/435705
[3.1.4.1]: ../../wiki/HiSeqX-Repeat-Expansion-Cohort
[3.1.4.2]: https://ega-archive.org/datasets/EGAD00001003562
[3.2.1]: https://www.ebi.ac.uk/ena/data/view/PRJEB3381
[3.2.2]: https://basespace.illumina.com/s/2K7LqNG7Mt1h
[3.2.3]: https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001224.v1.p1
[3.6]: ../../wiki/Sample-Information#platinum-genomes
[4.1]: issues
[5.1]: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4695224/
[5.2]: http://genome.cshlp.org/content/27/1/157
[5.3]: https://www.nature.com/ng/journal/v49/n4/full/ng.3801.html
[5.4]: https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-015-1479-3
